src/protocols/secure_channel/CASESession.h

```diff
@@ -157,13 +157,16 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
 
     FabricIndex GetFabricIndex() const { return mFabricIndex; }
 
+    bool IsPeerActive() { return mExchangeCtxt.HasValue() && mExchangeCtxt.Value()->HasReceivedAtLeastOneMessage(); };
     // Compute our Sigma1 response timeout.  This can give consumers an idea of
     // how long it will take to detect that our Sigma1 did not get through.
-    static System::Clock::Timeout ComputeSigma1ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig);
+    static System::Clock::Timeout ComputeSigma1ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig,
+                                                               bool isPeerActive);
 
     // Compute our Sigma2 response timeout.  This can give consumers an idea of
     // how long it will take to detect that our Sigma1 did not get through.
-    static System::Clock::Timeout ComputeSigma2ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig);
+    static System::Clock::Timeout ComputeSigma2ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig,
```

Comment:
Sigma2 is always a response.  So in this case the peer is always known "active".

Reply:
i agree, updated, thanks